base/applications/mscutils/eventvwr/eventvwr.c

```diff
@@ -3673,11 +3673,37 @@ WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
         case WM_INITMENUPOPUP:
             lParam = lParam;
             break;
+#endif
 
         case WM_CONTEXTMENU:
-            lParam = lParam;
+        {
+            RECT rc;
+            HTREEITEM hItem;
+            TVHITTESTINFO hInfo = {0};
+
+            INT xPos = GET_X_LPARAM(lParam);
+            INT yPos = GET_Y_LPARAM(lParam);
+
+            GetWindowRect(hwndTreeView, &rc);
+            hInfo.pt.x = xPos - rc.left;
+            hInfo.pt.y = yPos - rc.top;
+
+            if ((hItem = TreeView_HitTest(hwndTreeView, &hInfo)))
```

Comment:
```suggestion
            hItem = TreeView_HitTest(hwndTreeView, &hInfo);
            if (hItem)
```

Reply:
It's not the first time you've shown me this change. 
Hope next time I remember this

Reply:
That's about avoiding assignments within if or while conditions 😉